readme.md

```diff
@@ -80,6 +80,13 @@ Type: `Function`
 
 Receives stdout/stderr as the first argument and the unhook method as the second argument. Return a string to modify it. Optionally, when in silent mode, you may return a `boolean` to influence the return value of `.write(...)`.
 
+##### streams
+
+Type: `Array`, `stream.Writable`<br>
+Default: `[process.stdout, process.stderr]`
+
+[Writable streams](https://nodejs.org/api/stream.html#stream_writable_streams) to hook.
+
```

Comment:
Can you include a short description of some use-cases here? Like you commented on in the issue:

> The use case is to use a transform function to replace sensitive text from the console output. I also want the user to be able to customize the stdout and stderr used by the library and ultimately pass them to signale.

Sometimes it's not clear to users what they would use a certain option for, so it can be useful to include use-cases.